engines/sword2/resman.cpp

```diff
@@ -230,6 +230,9 @@ bool ResourceManager::init() {
 				return false;
 			}
 
+			// Use korean.clu instead of TEXT.CLU
+			if (_vm->_isKorTrs && !scumm_stricmp(_resFiles[i].fileName, "TEXT.CLU"))
+				strcpy_s(_resFiles[i].fileName, 20, "korean.clu");
```

Comment:
Please use `Common::strcpy_s(_resFiles[i].fileName, 20, "korean.clu")` so that Linux and Mac can build 🙂

Reply:
Thanks, I fixed it